apps/server/src/modules/tool/service/tool-validation.service.spec.ts

```diff
@@ -193,7 +193,69 @@ describe('ToolValidation', () => {
 		});
 	});
 
-	describe('validateUpdate', () => {
+	describe('validateUpdate is called', () => {
+		describe('when validating an oauth2 tool', () => {
+			it('should call the externalToolService.isOauth2Config', async () => {
```

Comment:
your structure seems inconsistent, the check for isOauth2Config being called is one time here on this level, and the other time deeper in the tree...